src/document-submit.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getService} from './service';
+
+/**
+ * @param {!Window} window
+ */
+export function installGlobalSubmitListener(window) {
+  submitHandlerFor(window);
+}
+
+/**
+ * @param {!Window} window
+ */
+export function uninstallGlobalSubmitListener(window) {
```

Comment:
We don't seem to use this?

Could we simplify this file and get rid of the service?